.rubocop.yml

```diff
@@ -75,17 +84,14 @@ Style/FormatString:
 Style/GuardClause:
   Enabled: false
 
-Style/HashSyntax:
-  EnforcedStyle: ruby19
```

Comment:
No need to duplicate the default configuration: https://github.com/bbatsov/rubocop/blob/v0.48.1/config/default.yml#L620-L621